Stratis.Bitcoin/Features/BlockStore/LoopSteps/BlockStoreInnerStepFindBlocks.cs

```diff
@@ -5,11 +5,21 @@ namespace Stratis.Bitcoin.Features.BlockStore.LoopSteps
 {
     /// <summary>
     /// Find blocks to download by asking the BlockPuller
-    /// 
-    /// Once the BatchDownloadSize has been reached
-    /// the task will be removed from the routine via context.StopFindingBlocks()
+    /// <para>
+    /// If a stop condition is found <see cref="ShouldStopFindingBlocks"/> and
```

Comment:
This seems like invalid use of `<see>`. What you see in IntelliSense here is "If a stop condition is found ShouldStopFindingBlocks and there arent blocks to download anymore return a Break() result causing the ..." which is probably not what you wanted.